model/execution/DeliveryExecutionList.php

```diff
@@ -125,9 +129,43 @@ private function createExecution($cachedData, $extraFields)
             $execution['online'] = $online;
         }
 
+        if ($isTimerAdjustmentAllowed) {
+            $reason = $this->getLastPauseReason($cachedData[DeliveryMonitoringService::DELIVERY_EXECUTION_ID]);
+            if ($reason) {
+                $execution['lastPauseReason'] = $reason;
+            }
+        }
+
         return $execution;
     }
 
+    private function getLastPauseReason(string $deliveryExecutionId): ?array
+    {
+        $reason = null;
+        $lastPause = $this->getDeliveryLogService()->search([
+            DeliveryLog::DELIVERY_EXECUTION_ID => $deliveryExecutionId,
+            DeliveryLog::EVENT_ID => DeliveryLogEvent::EVENT_ID_TEST_PAUSE,
+        ], [
+            'order' => 'created_at',
```

Comment:
I think we have auto increment ID in this table. Check indexes please if it wouldn't be better to order by ID.

Reply:
Not agree that usage of the identifiers is better than usage of dates, when I want to order by date. I've checked and we have an index on the created_at field